### ISSUE TITLE
Ensure buildbot runs upstream rpc-tests.

### ISSUE THREAD

**Issue by nathan-at-least** [Electric-Coin-Company/zcash#492]

**Comment by nathan-at-least** [comment]
Once these tests are running, we will have (some) automated verification of #267 and should close that ticket as we close this one.

**Comment by nathan-at-least** [comment]
We have a distinct but similar testing need for fetch-params testing (#682).  It's similar because it might be a long or bandwidth intensive test that we don't want to block every push-test on, but we *do* still want to regularly test it.

**Comment by defuse** [comment]
The command to run is:

```
qa/pull-tester/rpc-tests.py
```

**Comment by defuse** [comment]
It looks like this was stalling from indecision, so I went ahead and added a new column to the waterfall for the RPC test. 

There were two easy alternatives: add it to full-test-suite.sh, or make another waterfall column. I chose the latter since different waterfall columns get run in parallel within a single slave (builds on the same waterfall column aren't run in parallel within a slave), so adding it won't slow down the regular testing too much, and it lets us control when the two kinds of tests get run, separately. The only downside is that there's now a tiny bit of duplication in the buildbot config.
